logger.go

```diff
@@ -1,8 +1,8 @@
 package buffalo
 
 import (
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?